darwin/fixed.m

```diff
@@ -165,7 +162,7 @@ static void uiFixedSyncEnableState(uiDarwinControl *c, int enabled)
 uiDarwinControlDefaultHugsBottom(uiFixed, view)
 uiDarwinControlDefaultChildEdgeHuggingChanged(uiFixed, view)
 
-static void uiFixedChildVisibilityChanged(uiDarwinControl *c)
+static void uiuiprivFixedChildVisibilityChanged(uiDarwinControl *c)
```

Comment:
One "ui" too much

Reply:
Yeah I pushed so that I could check the changes in my VM. I'll fix that.